academy/4-my-own-chain/game-fifo.md

```diff
@@ -222,6 +241,202 @@ With these functions ready, it is time to use them in the message handlers.
 
 You implemented a FIFO that is updated but never really used.
 
+## Interact via the CLI
+
+Time to see what you get now with the commands. Because you made numerous additions to the blockchain state, you need to start afresh:
+
+```sh
+$ starport chain serve --reset-once
+```
+
+Don't forget to export `alice` and `bob` again as explained in an [earlier section](./create-message.md).
```

Comment:
I included this in a highlight box. But feel free to delete the box again or use a different one.